pkgs/top-level/default.nix

```
@@ -61,10 +61,23 @@ in let
   localSystem = lib.systems.elaborate args.localSystem;
 
   # Condition preserves sharing which in turn affects equality.
+  #
+  # See `lib.systems.equals` documentation for more details.
+  #
+  # Note that it is generally not possible to compare systems as given in
+  # parameters. E.g. system is initialized to
+  #
+  #   localSystem.system = "x86_64-linux";
+  #   crossSystem.config = "x86_64-unknown-linux-gnu";
```

```suggestion
  #   localSystem = { system = "x86_64-linux"; };
  #   crossSystem = { config = "x86_64-unknown-linux-gnu"; };
```
If you are mean those are the only fields of each argument set (because elaboration did not happen yet) I think this is clearer.

---

Done.